resident/resident-service/src/main/java/io/mosip/resident/service/impl/ResidentServiceImpl.java

```diff
@@ -1355,4 +1363,27 @@ public AidStatusResponseDTO getAidStatus(AidStatusRequestDTO reqDto) throws Resi
 		}
 	}
 
+	@Override
+	public String getCredentialRequestStatus(String aid) throws ResidentServiceCheckedException {
```

Comment:
Correct the property like `getCheckAidStatus`

Reply:
Done.